<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabwidgetRicercaPrenotazioneBici</class>
 <widget class="QTabWidget" name="tabwidgetRicercaPrenotazioneBici">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIcerca Prenotazione Bici</string>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Ricerca Prenotazione Bici</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnTornarePaginaPrecedente">
      <property name="text">
       <string>Tornare alla pagina precedente</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupboxSchedaRicercaBici">
      <property name="title">
       <string>Scheda ricerca bici:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QComboBox" name="comboboxInserireNumeroCamera">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>33</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>34</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>35</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>36</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnRicercaCamera">
         <property name="text">
          <string>Ricerca Camera</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelNumeroCamera">
         <property name="text">
          <string>Inserire il numero di camera del cliente:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupboxSchedaPrenotazioniAssociateCamera">
         <property name="title">
          <string>Scheda di tutte prenotazioni di bici associate alla camera ricercata:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <widget class="QTreeWidget" name="treeWidget">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Numero</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tipo</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Bici</string>
             </property>
             <property name="text">
              <string>UomoSenzaSeggiolino/DonnaConSeggiolino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bici</string>
             </property>
             <property name="text">
              <string>UomoSenzaSeggiolino/DonnaConSeggiolino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bici</string>
             </property>
             <property name="text">
              <string>UomoSenzaSeggiolino/DonnaConSeggiolino</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnConfermaScelteBici">
            <property name="text">
             <string>Conferma selezione</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
